### ISSUE TITLE
Add kubeconfig flag to load cluster configuration from different path

### ISSUE THREAD

**Issue by grafvonb** [NTTDATA-DACH/viewnode#10]
The configuration is not always present in the ~/.kube folder. It is common to specify a different path for the configuration.

**Comment by rothgar** [comment]
Would also be nice if it honored KUBECONFIG environment variable

**Comment by grafvonb** [comment]
Yes, this should also be implemented also. I add it to this issue. Thank you for your suggestion!

**Comment by grafvonb** [comment]
> Would also be nice if it honored KUBECONFIG environment variable

@rothgar implemented in 0.8.2 :) Thanks for the comment and enjoy it!
